ruby/webapp.rb.in

```diff
@@ -1,5 +1,20 @@
 plugin_path = "@CMAKE_SOURCE_DIR@/public" 
 
+#load dirs containing scripts
```

Comment:
Not the point. My point is: either install all of it, or none of it. This "install a plugin but not the JS" makes things really difficult to track.

If you made it a Ruby package, you would not need any Rake since the path to the local file can be accessed with __FILE__.